mender/install_mender.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+set -e
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run with sudo"
+  exit
+fi
+
+
```

Comment:
I'd use this export to define the base path for the files.

```suggestion
export SCRIPT_PATH=$(dirname $(readlink -f $0))
```